src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4403,6 +4403,60 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
+  /api/repository/v1/datasets/{id}/snapshotBuilder/count:
+    post:
+      tags:
+        - datasets
+        - repository
+      description: >
+        Given a list of snapshot builder cohorts, return the unique participant count for this dataset.
+      operationId: getSnapshotBuilderCount
```

Comment:
I don't see a `getSnapshotBuilderCount` method...should this be `getCountResponse`?

Reply:
Nevermind! I see it now 🤦‍♀️